jason_rolle/primitive_data_challenges/Tempature.py

```diff
@@ -0,0 +1,17 @@
+####step 1
+celsius_100 = (100-32)*(5/9)
+#Converting farenheit to celsius
+print(celsius_100)
+##float
+print(type(celsius_100))
+celsius_0= (0-32)*(5/9)
+##Converting farenheit to celsius
+print(celsius_0)
+print((34.2-32)*(5/9))
+##Converting farenheit to celsius
+print((5*9/5)+(32))
+##Converting celsius to farenheit
+print((30.2*9/5)+(32))
+print((85.1-32)*(5/9))
```

Comment:
`print((85.1-32)*(5/9))` converts 85.1 to celsius. The challenge asks to compare which is hotter, 30.2 celsius OR 85.1 farenheit. So in order to compare, for example, apples to apples, you will need to compare farenheit to farenheit. Which you already do in this print statement `print((85.1-32)*(5/9))`